src/app/modules/shared/components/details-fields/details-fields.component.ts

```diff
@@ -151,11 +153,14 @@ export class DetailsFieldsComponent implements OnChanges, OnInit {
     }
   }
 
-  cleanForm() {
+  cleanForm(skipProject: boolean = false): void {
     this.selectedTechnologies = [];
```

Comment:
Is necessary add this line  **this.technologies.query = '';** ?

I think with this line you can reset the technologies   **this.selectedTechnologies = [];**

Reply:
With the part of this.`selectedTechnologies =[]`; the tags that appear below the input are cleaned, but the `input` of technologies is not cleaned if there is something in it, that is why it is necessary to add this line, I think it is because this part is another component `app-technologies`  and is not directly associated with the form.